core/src/main/java/org/mule/runtime/core/internal/retry/policies/SimpleRetryPolicy.java

```diff
@@ -71,7 +71,7 @@ public SimpleRetryPolicy(long frequency, int retryCount) {
     net.jodah.failsafe.RetryPolicy<Object> actingPolicy = new net.jodah.failsafe.RetryPolicy<>()
         .handleIf(shouldRetry)
         .withMaxRetries(count != RETRY_COUNT_FOREVER ? count : -1)
-        .withDelay(frequency)
+        .withDelay(frequency.isZero() ? ofMillis(1) : frequency)
```

Comment:
why this change? how does it changes behavior?

Reply:
jodah does not allow a 0 as a valid parameter value, however we do allow 0 values for the frequency param in our retry policy